<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>4</y>
      <width>941</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;For minting NFT you need do next steps:&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;https://nft.storage/manage/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#000000;&quot;&gt;1: Write your wallet&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;https://nft.storage/manage/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#000000;&quot;&gt;2: Set your network&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;3: &lt;a href=&quot;https://nft.storage/login/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Register an NFT.Storage account&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;https://nft.storage/login/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#000000;&quot;&gt;4: Upload files to NFT.Storage&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;https://nft.storage/manage/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#000000;&quot;&gt;5: Add link to media file to input field&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;https://nft.storage/manage/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#000000;&quot;&gt;6. Write contract name &lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;https://nft.storage/manage/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#000000;&quot;&gt;6: Add metadata&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;https://nft.storage/manage/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#000000;&quot;&gt;7: Add number of copy&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="Wallet_edit">
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Your Wallet</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Network">
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Your network</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="NFT_storage">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>NFT.Storage API KEY</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QLineEdit" name="mediadata">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>MediaData</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="data">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>File</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="contract">
           <property name="placeholderText">
            <string>Contract name</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLineEdit" name="metatitle">
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>Metadata-title</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="meta_description">
             <property name="placeholderText">
              <string>Metadata-description</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="meta_media">
             <property name="placeholderText">
              <string>Metadata-media</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="meta_media_hash">
             <property name="placeholderText">
              <string>Metadata-media_hash</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Meta_copies">
             <property name="placeholderText">
              <string>Metadata-copies</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Meta_issued">
             <property name="placeholderText">
              <string>Metadata-issued_at</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Metaexpiries">
             <property name="placeholderText">
              <string>Metadata-expires_at</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Metastarts">
             <property name="placeholderText">
              <string>Metadata-starts_at</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Metaupdated">
             <property name="placeholderText">
              <string>Metadata-updated_at</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Metaextra">
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>Metadata-extra</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Metareference">
             <property name="placeholderText">
              <string>Metadata-reference</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="meta_reference_hash">
             <property name="placeholderText">
              <string>Metadata-reference_hash</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Number of copies</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="mint">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>610</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mint NFT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>949</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>Meta_copies</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>744</x>
     <y>614</y>
    </hint>
    <hint type="destinationlabel">
     <x>670</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
